prime-router/src/main/kotlin/azure/ReportWriter.kt

```diff
@@ -0,0 +1,43 @@
+package gov.cdc.prime.router.azure
+
+import gov.cdc.prime.router.Report
+import gov.cdc.prime.router.common.BaseEngine
+import gov.cdc.prime.router.serializers.CsvSerializer
+import gov.cdc.prime.router.serializers.Hl7Serializer
+import java.io.ByteArrayOutputStream
+
+class ReportWriter {
+    companion object {
```

Comment:
Make this into an object class and remove companion object

Reply:
Done